docs/articles/documentation/guides/continuous-integration/circleci.md

```diff
@@ -6,139 +6,224 @@ redirect_from:
-      "testcafe": "*",
-      "testcafe-reporter-xunit": "*"
+    "testcafe": "*",
+    "testcafe-reporter-xunit": "*"
   }
 }
 ```
 
-The `test` command runs tests from the `tests` directory in the headless Chrome. Reports are saved in the xUnit format at `/tmp/test-results/res.xml`.
+For more information on how to configure a test run with the `testcafe` command, see [Command Line Interface](../../reference/command-line-interface.md).
+
+## Step 5 - View Test Results
+
+Commit the changes to trigger the build. CircleCI Dashboard shows the results in the pipeline.
+
+Click the job name to see the details.
+
+![View the pipeline in the Dashboard](../../../images/circle-ci/view-pipeline.png)
+
+The `Steps` panel displays information about the completed job including the run duration, latest commit, and build log:
 
-Finally, push all changes to GitHub.
+![Job details in the Dashboard](../../../images/circle-ci/view-steps.png)
 
-## Step 3 - Running the Build
+The `Tests` panel displays test results. If errors occur, the report is shown.
 
-Use the **Start building** button to trigger a build.
+![View steps results in the Dashboard](../../../images/circle-ci/view-results.png)
 
-![Starting Build](../../../images/circle-ci/start-building.png)
+## Use Alternative Images
 
-## Step 4 - Viewing Test Results
+This guide uses the `cimg/node:lts-browsers`, an official CircleCI Docker image that comes with `lts` version of node and browser dependencies pre-installed. This image is in beta. If you run into problems with it, use one of the images listed below.
 
-Go to the **Builds** section and choose a build to view its results.
+### Use a TestCafe Image
 
-![Test Results](../../../images/circle-ci/fail-report.png)
+TestCafe is available as a preconfigured Docker image. The image has TestCafe, Chromium, and Firefox pre-installed.
+
+To use this image, paste the following contents in your `config.yml` on [Step 3](###-step-3---configure-the-build):
+
+```yml
+version: 2.1
+orbs:
+  node: circleci/node@4.1.0
+jobs:
+  test:
+    docker:
+      - image: testcafe/testcafe:latest
+    steps:
+      - checkout
+      - node/install-packages
+      - run:
+          command: npm run test
+      - store_test_results:
+          path: /tmp/test-results
+workflows:
+  e2e-test:
+    jobs:
+      - test
+```
+
+This image has no Google Chrome installed. To run tests in Chromium instead, paste the following code to your `package.json` on [Step 4](#step-4---add-the-`test`-script-to-package.json):
+
+```json
+  "scripts": {
+    "test": "testcafe chromium:headless, firefox:headless tests/**/* -r xunit:/tmp/test-results/testcafe/results.xml"
+  }
+```
+
+For more information about TestCafe Docker image, read [Use TestCafe Docker Image](../advanced-guides/use-testcafe-docker-image.md).
+
+### Use a Legacy CircleCI Image
+
+This example uses `circleci/node:lts-browsers`, a legacy image with `lts` version of Node, Google Chrome, and Firefox pre-installed.
```

Comment:
In the previous section, you said: "This guide" meaning the previous part of the guide. Here you are saying "This example" meaning a subsequent example.

Maybe change to "The following example"?

Maybe it's minor, but it really confused me...

Reply:
> The following example uses ...